### PR TITLE
Fix broken source links

### PR DESCRIPTION
**Pull request by fvisin**
This PR fixes the broken source button links and a couple of inconsistencies I found in the documentation.

### CODE REVIEW

File: doc/library/gof/fgraph.txt [hunk 1]
```diff
@@ -5,38 +5,40 @@
 :mod:`fgraph` -- Graph Container [doc TODO]
```
Comment (nouiz) [inline doc/library/gof/fgraph.txt]: rename this at the same time to fg.
fgraph -> fg

File: doc/library/gof/fgraph.txt [hunk 2]
```diff
@@ -5,38 +5,40 @@
...
 
 
-Guide
-=====
+.. _fgraph:
 
 FunctionGraph
 -------------
 
+.. class:: FunctionGraph
```
Comment (nouiz) [inline doc/library/gof/fgraph.txt]: I would add an autoclass of FunctionGraph as in the removed sandboxed version.

File: theano/gof/graph.py [hunk 3]
```diff
@@ -1246,9 +1242,9 @@ def as_string(i, o,
         Input `Variable` s.
     o : list
         Output `Variable` s.
-    leaf_formatter : function
+    leaf_formatter : theano.function
```
Comment (nouiz) [inline theano/gof/graph.py]: this isn't right.
theano.function -> callable

idem bellow.

**Comment by slefrancois** [comment]
test this please
testing jenkins

**Comment by slefrancois** [comment]
jenkins test this please

**Comment by fvisin** [comment]
I implemented the changes you requested, let's see how jenkins reacts to this PR.

**Comment by fvisin** [comment]
@nouiz I didn't change this line to `callable` yet:
https://github.com/Theano/Theano/blob/0c57414e5dea952655c346fc91e8712185ff77c3/theano/gof/graph.py#L1246.

I can do a commit directly to the repo (without a PR) to fix that if you agree.

**Comment by fvisin** [comment]
Fixed in 6b8d3f0

**Pull request merged**